packages/plugin/src/theia.d.ts

```diff
@@ -5250,6 +5353,15 @@ export module '@theia/plugin' {
          */
         export function createOutputChannel(name: string): OutputChannel;
 
+        /**
+         * Creates a new {@link OutputChannel output channel} with the given name.
+         * If options are given, creates a new {@link OutputChannel output channel} with the given name.
```

Comment:
This does not make much sense here. `options` is mandatory, and it will create a `LogOutputChannel` instance and not an `OutputChannel`. What about copying the docs from Code:

> Creates a new [log output channel](https://code.visualstudio.com/api/references/vscode-api#LogOutputChannel) with the given name.